rs/sns/swap/tests/swap.rs

```diff
@@ -3939,19 +3943,23 @@ fn test_create_sns_neuron_recipes_skips_already_created_neuron_recipes_for_nf_pa
         neurons_fund_participation_icp_e8s: Some(100 * E8),
         cf_participants: vec![
             CfParticipant {
-                hotkey_principal: i2principal_id_string(1001),
+                controller: Some(PrincipalId::new_user_test_id(1001)),
+                hotkey_principal: PrincipalId::new_user_test_id(1001).to_string(),
                 cf_neurons: vec![CfNeuron {
                     nns_neuron_id: 1,
                     amount_icp_e8s: 50 * E8,
                     has_created_neuron_recipes: Some(true),
+                    hotkeys: Some(Vec::new().into()),
                 }],
             },
             CfParticipant {
-                hotkey_principal: i2principal_id_string(1002),
+                controller: Some(PrincipalId::new_user_test_id(1002)),
+                hotkey_principal: PrincipalId::new_user_test_id(1002).to_string(),
                 cf_neurons: vec![CfNeuron {
                     nns_neuron_id: 2,
                     amount_icp_e8s: 50 * E8,
                     has_created_neuron_recipes: Some(false),
+                    hotkeys: Some(Vec::new().into()),
```

Comment:
We agreed to use From instead of Into

Reply:
Is that decision still serving us well?

Reply:
Not sure but I changed it anyway for clarity